src/helpers/flow.ts

```diff
@@ -0,0 +1,116 @@
+// Copied from
+// https://github.com/gcanti/fp-ts/blob/master/src/function.ts#L230
+/**
+ * @function
+ * @since 2.11.5
+ */
+export function flow<A extends ReadonlyArray<unknown>, B>(ab: (...a: A) => B): (...a: A) => B;
```

Comment:
Originally, this was called pipe and was copied from an older version of fp-ts where pipe was defined as flow.